Libraries/LibLine/XtermSuggestionDisplay.cpp

```diff
@@ -62,8 +64,45 @@ void XtermSuggestionDisplay::display(const SuggestionManager& manager)
         lines_used += max_line_count;
         longest_suggestion_length = 0;
     }
+
     VT::move_absolute(max_line_count + m_origin_x, 1);
 
+    if (m_pages.size() == 0) {
+        size_t num_printed = 0;
+        size_t lines_used { 1 };
```

Comment:
```suggestion
        size_t lines_used = 1;
```

for trivially constructed locals is the Serenity style.

Reply:
Noted.